pkg/model/revision.go

```diff
@@ -3,7 +3,7 @@ package model
 import (
 	"time"
 
-	"github.com/puppetlabs/nebula-sdk/pkg/workflow/spec/serialize"
+	"github.com/puppetlabs/relay-core/pkg/expr/serialize"
```

Comment:
I'm _pretty_ sure you can just use the Horsehead `transfer.JSONInterface` in place of `serialize.JSONTree` below without consequence and it would remove the dependency on `relay-core`.